? ggzd/game_servers/libggzmod/Makefile
? ggzd/game_servers/libggzmod/Makefile.in
? ggzd/game_servers/libggzmod/.libs
? ggzd/game_servers/libggzmod/.deps
? ggzd/game_servers/libggzmod/ggz.lo
? ggzd/game_servers/libggzmod/libggzdmod.la
Index: ggzd/configure.in
===================================================================
RCS file: /cvsroot/ggz/ggzd/configure.in,v
retrieving revision 1.63
diff -r1.63 configure.in
314,320d313
< 		game_servers/krosswater/Makefile
< 		game_servers/krosswater/stackpath/Makefile
< 		game_servers/krosswater/zoneserver/Makefile
< 		game_servers/muehle/Makefile
< 		game_servers/muehle/ggz/Makefile
< 		game_servers/muehle/qtserv/Makefile
< 		game_servers/muehle/variants/Makefile
Index: ggzd/game_servers/Makefile.am
===================================================================
RCS file: /cvsroot/ggz/ggzd/game_servers/Makefile.am,v
retrieving revision 1.21
diff -r1.21 Makefile.am
6c6
< 	  escape ggzcards hastings  krosswater lapocha spades muehle keepalive
---
> 	  escape ggzcards hastings  lapocha spades keepalive
Index: ggzd/ggzd/net.c
===================================================================
RCS file: /cvsroot/ggz/ggzd/ggzd/net.c,v
retrieving revision 1.45
diff -r1.45 net.c
123a124
> static void _net_handle_tls_start(GGZNetIO *net, GGZXMLElement *data);
221a223
> 		ggz_tls_disable_fd(net->fd);
259c261,262
< 	_net_send_line(net, "\t<SERVER ID='GGZ-%s' NAME='%s' VERSION='%d' STATUS='%s'>", VERSION, xml_srv_name, GGZ_CS_PROTO_VERSION, "ok");
---
> 	_net_send_line(net, "\t<SERVER ID='GGZ-%s' NAME='%s' VERSION='%d' STATUS='%s' TLS_SUPPORT='%s'>", VERSION, xml_srv_name, GGZ_CS_PROTO_VERSION, "ok",
> 		ggz_tls_support_query() ? "yes" : "no");
743c746
< 	if ( (len = read(net->fd, buf, BUFFSIZE)) < 0) {
---
> 	if ( (len = ggz_tls_read(net->fd, buf, BUFFSIZE)) < 0) {
895a899,900
> 	else if (strcmp(tag, "TLS_START") == 0)
> 		process_func = _net_handle_tls_start;
1459a1465,1470
> /* Function for <TLS_START> tag */
> static void _net_handle_tls_start(GGZNetIO *net, GGZXMLElement *data)
> {
> 	ggz_tls_enable_fd(net->fd, GGZ_TLS_SERVER);
> }
> 
1572c1583
< 	return write(net->fd, buf, strlen(buf));
---
> 	return ggz_tls_write(net->fd, buf, strlen(buf));
1584c1595
< 	return write(net->fd, buf, strlen(buf));
---
> 	return ggz_tls_write(net->fd, buf, strlen(buf));
